sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/Embedding.java

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.implementation.Constants;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Embedding settings within {@link VectorEmbeddingPolicy}
+ */
+public final class Embedding {
```

Comment:
Instead of just calling out Embedding, which is too generic of a name, shall we call it something else like - 
`VectorEmbedding` or `CosmosVectorEmbedding`? Though we try not to use `Cosmos` prefix.

Same goes for `DistanceFunction`

Reply:
+1 for both - and in this case using Comsos prefix absolutely makes sense - since every single database is adding vector stuff now :-)

Reply:
Update to CosmosVectorEmbedding, and CosmosVectorDistanceFunction, CosmosVectorDataType